WebApplication/ClientApp/src/components/downloads.js

```diff
@@ -129,7 +129,7 @@ export class Downloads extends Component {
                 // register the row
                 data.push({
                     id: 'bom',
-                    icon: 'products-and-services-24.svg',
+                    icon: 'file-spreadsheet-24.svg',
```

Comment:
Maybe you can use `FileSpreadsheet24` icon from @hig/icons package? (https://github.com/Autodesk/hig/blob/development/packages/icons/src/icons/information-dense/file-spreadsheet.svg)

Reply:
this icon is recommended by Martin